src/Analyzers/Analyzers/test/Microsoft.AspNetCore.Analyzers.Test.csproj

```diff
@@ -4,8 +4,13 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <RootNamespace>Microsoft.AspNetCore.Analyzers</RootNamespace>
+
+    <!-- Tests do not work on Helix or when bin/ directory is not in project directory due to undeclared dependency on test content. -->
     <!-- https://github.com/aspnet/AspNetCore/issues/6549 -->
     <BuildHelixPayload>false</BuildHelixPayload>
+    <BaseOutputPath />
+    <OutputPath />
```

Comment:
Do we have an issue or issues tracking fixing the tests and removing these workarounds?

Reply:
They are sort of generally tracked by https://github.com/aspnet/AspNetCore/issues/4248. Fixing tests which don't work on Helix will -- in most cases -- also solve this problem. The root cause of tests not working is the same - the test binary expects some files to exist in a relative location to the test binary.